src/LegendLanguageClient.ts

```diff
@@ -139,9 +165,273 @@ export class LegendLanguageClient extends LanguageClient {
     token?: CancellationToken,
   ): Promise<string> {
     if (token) {
-      return this.sendRequest(LEGEND_WRITE_ENTITY, request, token);
+      return this.sendRequest(LEGEND_WRITE_ENTITY_REQUEST_ID, request, token);
+    } else {
+      return this.sendRequest(LEGEND_WRITE_ENTITY_REQUEST_ID, request);
+    }
+  }
+
+  async getClassifierPathMap(token?: CancellationToken): Promise<string> {
+    if (token) {
+      return this.sendRequest(GET_CLASSIFIER_PATH_MAP_REQUEST_ID, token);
     } else {
-      return this.sendRequest(LEGEND_WRITE_ENTITY, request);
+      return this.sendRequest(GET_CLASSIFIER_PATH_MAP_REQUEST_ID);
     }
   }
+
+  async getSubtypeInfo(token?: CancellationToken): Promise<string> {
+    if (token) {
+      return this.sendRequest(GET_SUBTYPE_INFO_REQUEST_ID, token);
+    } else {
+      return this.sendRequest(GET_SUBTYPE_INFO_REQUEST_ID);
+    }
+  }
+
+  async analyzeMappingModelCoverage(
+    mappingFilePath: string,
+    mappingId: string,
+  ): Promise<string> {
+    return commands.executeCommand(
+      LEGEND_COMMAND,
+      mappingFilePath,
+      0,
```

Comment:
@rafaelbey do you know if it's safe to set the section index to 0 for all of these calls? Or is there a way we can get the section index from the file path and the entity name?

Reply:
Zero is not good... this assume one entity per file.  Need to check what exists to get the proper section.

Reply:
Have a solution to get the correct section index but it depends on this LSP change: https://github.com/finos/legend-engine-ide-lsp/pull/257

Reply:
Spoke with @rafaelbey and we will be updating the LSP's executeCommand function to no longer require section index.